src/utils/workspaceUtils.ts

```diff
@@ -44,3 +44,18 @@ export async function getActiveFilePath(uri?: vscode.Uri): Promise<string | unde
 export function getWorkspaceConfiguration(): vscode.WorkspaceConfiguration {
     return vscode.workspace.getConfiguration("leetcode");
 }
+
+function getHttpAgent(): string | undefined {
+    return vscode.workspace.getConfiguration("http").get<string>("proxy");
+}
+
+// clone process.env and add http proxy
+export function createEnvOption(): {} {
```

Comment:
Looks like cpUtils.ts is a better place to put this method?

Reply:
Sure. I'll move it there.